api/database/submodels/quest.py

```diff
@@ -0,0 +1,73 @@
+import bleach
```

Comment:
Just a quick comment - since we're not using the submodels right now we shouldn't merge these into main.

Reply:
You got it! Done and Done!!!